bundles/io/org.eclipse.smarthome.io.rest.core/META-INF/MANIFEST.MF

```diff
@@ -1,71 +1,72 @@
-Manifest-Version: 1.0
-Bundle-Activator: org.eclipse.smarthome.io.rest.core.internal.RESTCoreAc
- tivator
-Bundle-ManifestVersion: 2
-Bundle-Name: Eclipse SmartHome Core REST API
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-SymbolicName: org.eclipse.smarthome.io.rest.core
-Bundle-Vendor: Eclipse.org
-Bundle-Version: 0.10.0.qualifier
-Export-Package: 
- org.eclipse.smarthome.io.rest.core.config,
- org.eclipse.smarthome.io.rest.core.item,
- org.eclipse.smarthome.io.rest.core.persistence,
- org.eclipse.smarthome.io.rest.core.service,
- org.eclipse.smarthome.io.rest.core.thing
-Import-Package: 
- com.google.common.collect,
- com.google.gson,
- io.swagger.annotations;resolution:=optional,
- javax.annotation.security;resolution:=optional,
- javax.ws.rs,
- javax.ws.rs.core,
- javax.ws.rs.ext,
- org.apache.commons.lang,
- org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.smarthome.config.core,
- org.eclipse.smarthome.config.core.dto,
- org.eclipse.smarthome.config.core.status,
- org.eclipse.smarthome.config.core.validation,
- org.eclipse.smarthome.config.discovery,
- org.eclipse.smarthome.config.discovery.dto,
- org.eclipse.smarthome.config.discovery.inbox,
- org.eclipse.smarthome.core.auth,
- org.eclipse.smarthome.core.binding,
- org.eclipse.smarthome.core.binding.dto,
- org.eclipse.smarthome.core.common,
- org.eclipse.smarthome.core.common.registry,
- org.eclipse.smarthome.core.events,
- org.eclipse.smarthome.core.extension,
- org.eclipse.smarthome.core.i18n,
- org.eclipse.smarthome.core.items,
- org.eclipse.smarthome.core.items.dto,
- org.eclipse.smarthome.core.items.events,
- org.eclipse.smarthome.core.library.items,
- org.eclipse.smarthome.core.library.types,
- org.eclipse.smarthome.core.persistence,
- org.eclipse.smarthome.core.persistence.dto,
- org.eclipse.smarthome.core.thing,
- org.eclipse.smarthome.core.thing.binding.firmware,
- org.eclipse.smarthome.core.thing.dto,
- org.eclipse.smarthome.core.thing.firmware,
- org.eclipse.smarthome.core.thing.firmware.dto,
- org.eclipse.smarthome.core.thing.i18n,
- org.eclipse.smarthome.core.thing.link,
- org.eclipse.smarthome.core.thing.link.dto,
- org.eclipse.smarthome.core.thing.profiles,
- org.eclipse.smarthome.core.thing.type,
- org.eclipse.smarthome.core.thing.util,
- org.eclipse.smarthome.core.transform;resolution:=optional,
- org.eclipse.smarthome.core.types,
- org.eclipse.smarthome.io.rest,
- org.eclipse.smarthome.io.rest.core.config,
- org.eclipse.smarthome.io.rest.core.item,
- org.eclipse.smarthome.io.rest.core.persistence,
- org.eclipse.smarthome.io.rest.core.service,
- org.eclipse.smarthome.io.rest.core.thing,
- org.osgi.framework,
- org.osgi.service.cm,
- org.osgi.service.component,
- org.slf4j
-Service-Component: OSGI-INF/*.xml
+Manifest-Version: 1.0
```

Comment:
Can you please check your formatter settings? The diff says you have changed the whole file although you probably have removed just one dependency.

Reply:
I have the default formatter settings, haven't really tweaked anything there. Not sure why this is happening: now I pulled the file again from the `master`, changed the single line I needed (for now, just adding the `...io.json` bundle as a dependency, and it still seems to show up like that.